subtitle.pm

```diff
@@ -0,0 +1,70 @@
+# 2015 by Bernhard M. Wiedemann
+# Licensed under GPL v2 or later
```

Comment:
work time => (c) SUSE Linux GmbH or SUSE LLC. Reminds me that os-autoinst needs proper license headers in all files.